src/Razor/src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/VisualStudioWindowsSolutionStatusService.cs

```diff
@@ -1,20 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using System.ComponentModel.Composition;
-using System.Diagnostics.CodeAnalysis;
-using Microsoft.VisualStudio.Editor.Razor;
-
-namespace Microsoft.VisualStudio.Mac.LanguageServices.Razor
-{
-    [Export(typeof(RazorSolutionStatusService))]
-    internal class VisualStudioWindowsSolutionStatusService : RazorSolutionStatusService
```

Comment:
lmao, just added this and realized I messed up the name too given it's in the mac project